README.md

```diff
@@ -32,6 +32,8 @@ Current features:
 
 - Using IDE built-in plugin system:
   
+  Be careful, the plugin might be not up to date!
```

Comment:
Why

Reply:
Because I started with a plug-in on version 0.0.8 and it wouldn’t update and the IDEA page told me, that your plug in is in review. What I want to emphasize is: if you download the plugin from GitHub it is for sure the latest 😅

Reply:
I would more complete the sentence like All updates needs to be checked  Jetbrains and this can take 1-2 days. If you want to download it earlier use the artifact from the Github release

Reply:
If it is only 1-2 days I think it is not needed.